aries-site/src/examples/templates/SidebarExample.js

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Box, Button } from 'grommet';
+import {
+  Apps,
+  Chat,
+  Clock,
+  Configure,
+  Terminal,
+  ShieldSecurity,
+  StatusUnknown,
+} from 'grommet-icons';
+
+export const SidebarExample = ({ direction, mobile, ...rest }) => (
+  <Box
+    direction={direction}
+    background="blue"
```

Comment:
We should get clarification from @L0ZZI on the color here. The designs show the same #0E5265 (blue.dark) for the sidebar background regardless of light or dark context.

https://www.figma.com/file/JdmovKzBcjU7e4NoFCcxcE/hpe-common-cx-navigation?node-id=60%3A170

![Screen Shot 2020-02-20 at 1 21 25 PM](https://user-images.githubusercontent.com/1756948/74975201-eac9d980-53e3-11ea-9474-36b5946c9759.png)